bazel/repositories.bzl

```diff
@@ -0,0 +1,191 @@
+        sha256 = "eb1ab04781474360f77c318ab89d8c5a03abc38e63d65a603cabbf1b00a1dc90",
+        strip_prefix = "openssl-3.0.9",
+        url = "https://vectorized-public.s3.us-west-2.amazonaws.com/dependencies/openssl-3.0.9.tar.gz",
+    )
+
+    http_archive(
+        name = "rapidjson",
+        build_file = "//bazel/thirdparty:rapidjson.BUILD",
+        sha256 = "d085ef6d175d9b20800958c695c7767d65f9c1985a73d172150e57e84f6cd61c",
+        strip_prefix = "rapidjson-14a5dd756e9bef26f9b53d3b4eb1b73c6a1794d5",
+        url = "https://github.com/redpanda-data/rapidjson/archive/14a5dd756e9bef26f9b53d3b4eb1b73c6a1794d5.tar.gz",
+    )
+
+    http_archive(
+        name = "roaring",
+        build_file = "//bazel/thirdparty:roaring.BUILD",
+        sha256 = "78487658b774f27546e79de2ddd37fca56679b23f256425d2c86aabf7d1b8066",
+        strip_prefix = "CRoaring-c433d1c70c10fb2e40f049e019e2abbcafa6e69d",
+        url = "https://github.com/redpanda-data/CRoaring/archive/c433d1c70c10fb2e40f049e019e2abbcafa6e69d.tar.gz",
+    )
+
+    http_archive(
+        name = "seastar",
+        build_file = "//bazel/thirdparty:seastar.BUILD",
+        sha256 = "88d71ffe853fdffef4669009367457ac5ea461a4afb688fa74528549520244ce",
+        strip_prefix = "seastar-6a4727efd9154ab475860578f01defcbb95df730",
+        url = "https://github.com/redpanda-data/seastar/archive/6a4727efd9154ab475860578f01defcbb95df730.tar.gz",
+    )
+
+    http_archive(
+        name = "snappy",
+        build_file = "//bazel/thirdparty:snappy.BUILD",
+        sha256 = "774c337a545d6b818163c12455627ba61ed8d2daa9a9236b3aa88c64f081560e",
+        strip_prefix = "snappy-ca541bd6c80d97cf93d1a33e613521947701ea53",
+        url = "https://github.com/redpanda-data/snappy/archive/ca541bd6c80d97cf93d1a33e613521947701ea53.tar.gz",
+    )
+
+    http_archive(
+        name = "unordered_dense",
+        build_file = "//bazel/thirdparty:unordered_dense.BUILD",
+        sha256 = "98c9d02ff8761d50a2cb6ebd53f78f7d311f6980aef509efdcdaa5f3868ca06c",
+        strip_prefix = "unordered_dense-9338f301522a965309ecec58ce61f54a52fb5c22",
+        url = "https://github.com/redpanda-data/unordered_dense/archive/9338f301522a965309ecec58ce61f54a52fb5c22.tar.gz",
+    )
+
+    http_archive(
+        name = "wasmtime",
+        build_file = "//bazel/thirdparty:wasmtime.BUILD",
+        sha256 = "a7f989b170d109696b928b4b3d1ec1d930064af7df47178e1341bd96e5c34465",
+        strip_prefix = "wasmtime-9e1084ffac08b1bf9c82de40c0efc1baff14b9ad",
+        url = "https://github.com/bytecodealliance/wasmtime/archive/9e1084ffac08b1bf9c82de40c0efc1baff14b9ad.tar.gz",
+    )
+
+    http_archive(
+        name = "xxhash",
+        build_file = "//bazel/thirdparty:xxhash.BUILD",
+        sha256 = "716fbe4fc85ecd36488afbbc635b59b5ab6aba5ed3b69d4a32a46eae5a453d38",
+        strip_prefix = "xxHash-bbb27a5efb85b92a0486cf361a8635715a53f6ba",
+        url = "https://github.com/Cyan4973/xxHash/archive/bbb27a5efb85b92a0486cf361a8635715a53f6ba.tar.gz",
```

Comment:
FWIW these hashes can change and it's kind of a pain to use bazel for these. We should probably mirror them, but this doesn't need to happen now.

Reply:
https://redpandadata.atlassian.net/browse/CORE-5573